### ISSUE TITLE
Grafana graph time resolution is limited to 1.0 seconds

### ISSUE THREAD

**Issue by bgottula** [bgottula/track#104]
I've been logging telemetry at a rate of 10 Hz for most tracking passes, yet my Grafana dashboard is displaying it as if I only have data points every 1.0 seconds. Fix this.

**Comment by bgottula** [comment]
Looks like sub-second time resolution support in Grafana may be spotty since the most common use-case for Grafana is apparently metrics from things like data centers that are rarely collected faster than 1 Hz. However in the query builder for each graph I was able to work-around the issue by modifying the "Group By" section by replacing `time($__interval)` with `time(100ms)`. I don't have a deep understanding of what Group By does or how `$__interval` is set but it appears that `$__interval` is supposed to scale itself based on the current span of the time axis in the graph, but apparently Grafana never sets it to less than `1s`. My approach using a hard-coded value might result in using excessively high resolution on the time axis for the query when the graph is zoomed out to very large timespans.

I applied these changes on nix but I have not saved or committed them yet.
